src/ui/scss/init/_gui.scss

```diff
@@ -12,7 +12,7 @@ html {
   background-color: $lbry-white;
   font-size: 12px;
   height: 100%;
-  overflow: hidden;
+  overflow-x: hidden;
```

Comment:
I tried to apply overflow for both axis (x / y), but I'll need to change `.main-wrapper ` css as well in order to work, see:  https://github.com/lbryio/lbry-desktop/pull/2521#issuecomment-501127146